crypto/pkcs12/p12_mutl.c

```diff
@@ -15,12 +15,18 @@
 
 #include <stdio.h>
 #include "internal/cryptlib.h"
+#include "crypto/evp.h"
 #include <openssl/crypto.h>
 #include <openssl/hmac.h>
 #include <openssl/rand.h>
 #include <openssl/pkcs12.h>
 #include "p12_local.h"
 
+static int pkcs12_pbmac1_pbkdf2_key_gen(const char *pass, int passlen,
+                                        unsigned char *salt, int saltlen,
+                                        int id, int iter, int keylen,
+                                        unsigned char *out,
+                                        const EVP_MD *md_type);
 int PKCS12_mac_present(const PKCS12 *p12)
```

Comment:
Blank line to separate the forward declaration for the function?

Reply:
Done, thanks! Probably this should be caught by the style checking script?

Reply:
Yes, `check-format.pl` should have caught this - did you try it out?

Reply:
No, I didn't.

I mean the script used for style check by tests - is it `check-format.pl` or smth else?

Reply:
It is that script.